libs/client/request-for-access/feature/src/lib/request-for-access.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'graduates-request-for-access-feature',
+  templateUrl: './request-for-access.component.html',
+  styleUrls: ['./request-for-access.component.scss'],
+})
+export class RequestForAccessComponent {
+  buttons: string[] = [];
+
+  constructor(){
+    for(let i=0; i<5; i++){
```

Comment:
```suggestion
  constructor() {
    for(let i=0; i<5; i++) {
```
We should probably run auto formatters on our code 😂